examples/hello_world_example.py

```diff
@@ -0,0 +1,7 @@
+from idom import component, html, run
```

Comment:
I don't really know that I need examples to be automatically added to a directory for me. Especially since these seem like they were just copy-pasted from files in the `docs/` directory.